schema.json

```diff
@@ -1581,6 +1581,22 @@
       },
       {
         "type": "imageDimension",
+        "label": "Thumbnail image in product page",

```

Comment:
I would place this one above the other thumbnail so both product page images are one after the other

Reply:
Also we might need to rename the label for the other thumbnail appropriately to specify where it will be used so it does not confuse people. Do we have a list of where it is used currently ?

Reply:
Good call on the reordering.

So, `productthumb_size` (the 100x100 one) appears to currently be used for the cart preview and the full cart page, as well as the orders page and returns page.  Possibly these could be summarized as a "Thumbnail image in cart and orders pages"?

It's used in one more place, though, and that's when a product has a product list picker as an option, as in the screenshot below. These display as just under 50x50 though, so I'm actually inclined to switch these over to the new 50x50 `productview_thumb_size`. Since these happen to appear in the product details page as well, the same description probably still works.

![screen shot 2017-10-11 at 11 27 09 am](https://user-images.githubusercontent.com/32500994/31459176-3c12ec88-ae77-11e7-8a4f-5e1d3d99f859.png)